src/olympia/devhub/tasks.py

```diff
@@ -369,6 +369,15 @@ def annotate_legacy_addon_restrictions(results, is_new_upload):
         insert_validation_message(
             results, message=msg, msg_id='thunderbird_and_seamonkey_migration')
 
+    # Legacy add-ons are no longer supported on AMO (if waffle is enabled).
+    elif (is_extension_or_complete_theme and
+            waffle.switch_is_active('disallow-legacy-submissions')):
+        msg = ugettext(
+            u'Legacy extensions are no longer supported in Firefox.')
```

Comment:
Should this link to our blog maybe, for extra guidance?

Reply:
Up to @jvillalobos - the issue didn't specify a link to include.

Reply:
We emailed the affected developers about it, so I don't think it'll be much use.